croud/__main__.py

```diff
@@ -232,6 +235,26 @@
                                ),
                                region_arg],
                 "calls": clusters_list,
+            },
+            "deploy": {
+                "help": "Deploys a new CrateDB cluster.",
+                "extra_args": [output_fmt_arg,
```

Comment:
Please use consistent list formatting:

```suggestion
                "extra_args": [
                    output_fmt_arg,
                    ...
                ]
```

Reply:
Note, that the black formatting is turned off for the command tree, to be able to make lists and dicts consistent and avoid bringing multiple arguments into a single line.
This is for better readability.